### ISSUE TITLE
[Android] Multiple selection images do not work in the basic gallery of Samsung Galaxy S10

### ISSUE THREAD

**Issue by LeeKyungJoon** [ivpusic/react-native-image-crop-picker#1475]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.35.1
- react-native v0.63.2

### Platform
Tell us to which platform this issue is related

- iOS 14.1
- Android 10

### Steps to reproduce

1.

2.

3.

### Attachments

Multiple selection images do not work in the basic gallery of Samsung Galaxy S10. 
In other places,multiple selection images working, but it doesn't work only in the basic gallery.
What should I do?

Love react-native-image-crop-picker? Please consider supporting our collective:
👉  https://opencollective.com/react-native-image-crop-picker/donate

**Comment by GOGOGOSIR** [comment]
me too, how can i resolve?

**Comment by nguyenductoan1995** [comment]
me too, how can resolve ?

**Comment by ssayebee** [comment]
`android/app/src/debug/AndroidMaifest.xml`
```xml
<manifest ...>
    <application ... android:requestLegacyExternalStorage="true" ... />
</manifest>
```

**Comment by VadimDyliko** [comment]
Looks like it does not work for all Samsung devices.

**Comment by maxime-memtell** [comment]
Yeah agreed it doesn't seem to do it in the Gallery of my Samsung A50. Somehow it works in the default interface it opens but not in the normal gallery. Process is quite painful for a user who wants to select multiple pictures from their gallery as a result -> click button to open picker -> navigate to gallery -> find picture -> repeat
Any solutions?

**Comment by bed-sb** [comment]
here is a patch thats needed to fix this, https://github.com/Shadowboxer-OpenSource/react-native-image-crop-picker/commit/c0c48fc4703ce0050a630b4beb1a1b4a1ed32e75

**Comment by jonno85** [comment]
> here is a patch thats needed to fix this, [Shadowboxer-OpenSource@c0c48fc](https://github.com/Shadowboxer-OpenSource/react-native-image-crop-picker/commit/c0c48fc4703ce0050a630b4beb1a1b4a1ed32e75)

Hi @bed-sb can't you open a PR on your fix?

**Comment by bed-sb** [comment]
Heyo! I can, I just don't have too much time to go back and forth with it atm (wrangling a 3yo and a 1 month old baby here!)

**Comment by jonno85** [comment]
> @jonno85 apologies for the delay, here's the PR #1779

No problems at all, @bed-sb thank you for the support 🙏
